### PR TITLE
Test all common rubies on Travis again.

### PR DESCRIPTION
**Pull request by splattael**
I had to move development dependencies to `Gemfile` allowing tests on jruby (`sqlite3` gem).

Deleting unused empty method `nested_accessible_attrs` fixed tests for 1.8 rubies.

### CODE REVIEW

**Comment by elight** [comment]
Huh. Not sure how I missed this.  I'll look at it shortly!

I was having an absolutely miserable afternoon, for personal reasons, when I started running into issues with JRuby and sqlite3.  I threw my arms up in the air in frustrations so I could get back into flow and attain a little peace.

Really appreciate it!  This was bothering me.  There's no reason it shouldn't be compatible with JRuby except for fighting with the dev env.

File: lib/redtape.rb [hunk 1]
```diff
@@ -17,9 +17,6 @@ def self.validates_and_saves(*args)
       self.model_accessors = args
     end
 
-    def self.nested_accessible_attrs(attrs = {})
-    end
```
Comment (elight) [inline lib/redtape.rb:21]: Yeah, this is dead code.  Removed it in the "automation" (current development) branch.  Was thinking too far ahead (YAGNI'ing) when pairing.

File: Gemfile [hunk 2]
```diff
@@ -2,3 +2,17 @@ source 'https://rubygems.org'
...
+  gem 'rails'
+
+  platform :ruby do
+    gem 'sqlite3'
+  end
+
+  platform :jruby do
+    gem 'activerecord-jdbcsqlite3-adapter'
+  end
+end
```
Comment (elight) [inline Gemfile:18]: Niiiice. Now I need to use RVM to test on all of these locally. Makes me glad I paired with Wayne on that back in the day (not that he needed the help!).

**Comment by elight** [comment]
Ok, I admit it: I was lazy. After fighting with RVM a little (testing in multiple Rubies doesn't have good/curent docs any more in RVM), I just merged your change and pushed.  Works like a champ. :D

**Comment by splattael** [comment]
Great, thanks for merging! :)

I'm not happy moving development dependencies from gemspec to Gemfile :-(
It just works but I'd love to know the "right way" without building platform specific gems.

**Comment by elight** [comment]
You and me both.  I haven't had to deal with multi-platform dev env dependencies for my gems in the past.    

Obviously, redtape won't care about the deployed environment.  That is, ActiveRecord is the piece that has the platform-specific support.  It's not our problem. :D

**Pull request merged**